packages/errors/src/errors.ts

```diff
@@ -351,6 +351,30 @@ export const AllCypressErrors = {
 
         https://on.cypress.io/run-group-name-not-unique`
   },
+  CLOUD_AUTO_CANCEL_NOT_AVAILABLE_IN_PLAN: (arg1: {link: string}) => {
+    return errTemplate`\
+      ${fmt.highlightSecondary(`Auto Cancellation`)} is not included under your current billing plan.
+      To enable this service, please visit your billing and upgrade to another plan with Auto Cancellation.
+      ${fmt.off(arg1.link)}`
+  },
+  CLOUD_AUTO_CANCEL_MISMATCH: (arg1: {runUrl: string}) => {
```

Comment:
Would this error trigger before the `CLOUD_PARALLEL_GROUP_PARAMS_MISMATCH` error on the cloud side?

Reply:
Yes! this would happen before `CLOUD_PARALLEL_GROUP_PARAMS_MISMATCH `